pkg/kubelet/config/file.go

```diff
@@ -34,20 +34,11 @@ import (
 	utilio "k8s.io/utils/io"
 )
 
-type podEventType int
-
 const (
-	podAdd podEventType = iota
-	podModify
-	podDelete
-
```

Comment:
All of the removed constants and `podEventType` are used in the code.

Reply:
Reverted my changes.